atomic_reactor/plugins/pre_add_help.py

```diff
@@ -66,6 +68,29 @@ def run(self):
             result['status'] = self.NO_HELP_FILE_FOUND
             return result
 
+        dockerfile = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
+        labels = dockerfile.labels
+
+        try:
+            help_file = open(help_path, 'r+')
```

Comment:
Maybe use `with open(help_path, 'r+') as help_file:` here, and remove the explicit `.close()` call?